### PR TITLE
Feat result page

### PR DESCRIPTION
**Pull request by krishanu2911**
Added the result page of the quiz app.
Screenshot (464)

### CODE REVIEW

File: pages/quiz-result.html [hunk 1]
```diff
@@ -4,9 +4,67 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="https://maple-ui.netlify.app/CSS/base.css">
+    <link rel="stylesheet" href="../CSS/quiz-category.css">
+    <link rel="stylesheet" href="../CSS/util.css">
+    <title>Result</title>
```
Comment (shuntzhere) [inline pages/quiz-result.html]: Is it possible to import all the CSS files to one main file? If so do that.

**Review by shuntzhere — COMMENTED** [review]
UI looks really nice, very easily understandable classnames. Suggested a minor change though.

**Comment by aishwarya-bm** [comment]
Clean &modular code with good class naming.
Same suggestion as above

**Review by aishwarya-bm — COMMENTED** [review]
Import all css in a main css file to reuse in all classes

File: pages/quiz-result.html (hunk 1 above)
Comment (krishanu2911) [reply pages/quiz-result.html]: Thank you for pointing out planning to do in future but update now only😃

**Review by aishwarya-bm — APPROVED** [review]

**Review by shuntzhere — APPROVED** [review]
Go ahead with the merge.

**Pull request merged**